pkg/kev/init.go

```diff
@@ -81,7 +80,7 @@ func (r *InitRunner) EnsureFirstInit() error {
 		return err
 	}
 
-	s.Success(time.Second * 1)
+	s.Success(0)
```

Comment:
makes sense for now I guess. Later we could have it managed centrally / or locally when needed by chaining `.Wait(time.Seconds * 1)` or similar. Could be done in a follow up PR as discussed.

Reply:
This is outdated.